plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixClient.java

```diff
@@ -804,7 +807,7 @@ private JdbcTypeHandle getArrayElementTypeHandle(JdbcTypeHandle arrayTypeHandle)
         String arrayTypeName = arrayTypeHandle.getJdbcTypeName()
                 .orElseThrow(() -> new TrinoException(PHOENIX_METADATA_ERROR, "Type name is missing for jdbc type: " + JDBCType.valueOf(arrayTypeHandle.getJdbcType())));
         checkArgument(arrayTypeName.endsWith(" ARRAY"), "array type must end with ' ARRAY'");
-        arrayTypeName = arrayTypeName.substring(0, arrayTypeName.length() - " ARRAY".length());
+        arrayTypeName = arrayTypeName.substring(0, arrayTypeName.length() - " ARRAY" .length());
```

Comment:
undo

Reply:
@s2lomon I think you might have something wrong with your formatter